grammar.js

```diff
@@ -1059,10 +1066,10 @@ module.exports = grammar({
 
     interpolation_format_clause: $ => seq(':', /[^}"]+/),
 
-    invocation_expression: $ => prec(PREC.INVOCATION, seq(
+    invocation_expression: $ => seq(
```

Comment:
It seems that you have accidentally removed the precedence value I added here, that is why the test is failing.